src/Faker/Provider/de_DE/Company.php

```diff
@@ -11,5 +11,14 @@ class Company extends \Faker\Provider\Company
         '{{lastName}}',
     );
 
+    /**
+     * @link http://www.personalseite.de/information/titel.htm
+     */
+    protected static $jobTitleFormat = array(
+        'Abteilungsdirektor', 'Arbeitsdirektor', 'Aufsichtsrat', 'Beirat', 'Bereichsleiter', 'Betriebsleiter', 'Finanzvorstand', 'Geschäftsführender Gesellschafter', 'Geschäftsführer', 'Gesellschafter',
+        'Handlungsbevollmächtigter', 'Kaufmännischer Vorstand', 'Leiter Rechtsabteilung', 'Mitglied des Aufsichtsrats', 'Personalleiter', 'Prokurist', 'Sellvertretender Vorsitzender des Vorstandes',
+        'Vorsitzender der Geschäftsführung', 'Vorsitzender des Aufsichtsrats', 'Vorsitzender des Vorstandes ', 'Vorstand Personal ', 'Vorstand Technik ', 'Vorstand Vertrieb ', 'Vorstandsmitglied ', 'Werksleiter'
```

Comment:
```suggestion
        'Vorsitzender der Geschäftsführung', 'Vorsitzender des Aufsichtsrats', 'Vorsitzender des Vorstandes ', 'Vorstand Personal ', 'Vorstand Technik', 'Vorstand Vertrieb', 'Vorstandsmitglied', 'Werksleiter'
```

Reply:
Now I see what you mean ;-) Sorry for the misunderstanding and thank you for the hint